README.md

```diff
@@ -420,6 +422,18 @@ Use periods to describe a path to nested properties. For example, the value `'co
 }
 ```
 
+If property names within the path include periods, you need to use an array of strings. For example, the value `['configs', 'foo.bar', 'baz']` will get you the `"baz"` value in a \`package.json like this:
```

Comment:
this is such an edge case that I doubt it matters, but with the additional comment maybe change this to

```
If **nested** property names within the path include periods
```

Reply:
Yes, this is a very ugly edge case. But sure I'll make that change.

Reply:
after that I think we're good then.  If people get confused for some reason we can amend wording later.

Thanks for coding this.